governance/roles.md

```diff
@@ -30,36 +38,36 @@ access. In any case, governance is not enforced by [access permissions](github.m
 but rather by members who are expected to thoughtfully consider their actions
 to support the group.
 
-## Role of members
+## Role of Members
```

Comment:
should all be sentence case as per contribution guide

Reply:
Done. Also updating other heads to follow the same rule.